service/controller/v1/key/key.go

```diff
@@ -1,18 +1,16 @@
 package key
 
 import (
-	"fmt"
-	"strings"
-
 	"github.com/giantswarm/apiextensions/pkg/apis/core/v1alpha1"
 	"github.com/giantswarm/microerror"
 )
 
 const (
-	LabelApp          = "app"
-	LabelManagedBy    = "giantswarm.io/managed-by"
-	LabelOrganization = "giantswarm.io/organization"
-	LabelServiceType  = "giantswarm.io/service-type"
+	LabelApp            = "app"
+	LabelManagedBy      = "giantswarm.io/managed-by"
+	LabelOrganization   = "giantswarm.io/organization"
+	LabelReleaseVersion = "release.giantswarm.io/version"
```

Comment:
We have to add the release version as label to the `ChartConfig` CRs in order to fetch the current state easily.

Reply:
Cool to have this label. I can see it being useful for lots of things.